rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/schedule/Sw360ScheduleService.java

```diff
@@ -42,4 +41,84 @@ public RequestStatus cancelAllServices(User sw360User) throws TException {
         }
     }
 
+    public RequestSummary scheduleCveSearch(User sw360User) throws TException{
+        String serviceName = ThriftClients.CVESEARCH_SERVICE;
+        try {
+            if (PermissionUtils.isUserAtLeast(UserGroup.ADMIN, sw360User)) {
+                RequestSummary requestSummary = new ThriftClients().makeScheduleClient().scheduleService(serviceName);
+                return requestSummary;
+            } else {
+                throw new HttpMessageNotReadableException("User is not admin");
+            }
+        } catch (TException e) {
+            throw new TException(e.getMessage());
+        }
+    }
+
+    public RequestStatus cancelCveSearch(User sw360User) throws TException{
+        String serviceName = ThriftClients.CVESEARCH_SERVICE;
+        try {
+            if (PermissionUtils.isUserAtLeast(UserGroup.ADMIN, sw360User)) {
+                RequestStatus requestStatus = new ThriftClients().makeScheduleClient().unscheduleService(serviceName, sw360User);
+                return requestStatus;
+            } else {
+                throw new HttpMessageNotReadableException("User is not admin");
```

Comment:
This seems to be not correct response code. 403 sounds appropriate here, but in this case it will throw the 400- Bad request?

Ref: https://github.com/eclipse-sw360/sw360/blob/ee3ed068411f42470cdaed148485cb96565f831b/rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/RestExceptionHandler.java#L43

Reply:
done.